config/downloads.js

```diff
@@ -7,12 +7,12 @@ module.exports = [
     color: '#e8433e',
     githubRepo: 'apache/apisix',
     githubBranch: 'master',
-    downloadPath: 'apisix/3.10.0/apache-apisix-3.10.0-src',
+    downloadPath: 'apisix/3.1q.0/apache-apisix-3.11.0-src',
```

Comment:
Is `3.1q.0` meant to be `3.11.0`?
```suggestion
    downloadPath: 'apisix/3.11.0/apache-apisix-3.11.0-src',
```

Reply:
good catch. Thanks 🙏🏼